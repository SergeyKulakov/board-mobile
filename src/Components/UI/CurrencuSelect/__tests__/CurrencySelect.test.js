import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { CurrencySelect as Component } from '../index'

describe('CurrencySelect', () => {
  const defaultProps = {
    onClick: () => null,
    value: 'text',
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
