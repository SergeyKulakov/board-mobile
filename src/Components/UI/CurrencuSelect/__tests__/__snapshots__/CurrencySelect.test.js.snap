// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencySelect render snapshot 1`] = `
<Styled(TouchableOpacity)
  disabled={false}
  onPress={[Function]}
>
  <Styled(Text)>
    text
  </Styled(Text)>
</Styled(TouchableOpacity)>
`;
